hyperframe/frame.py

```diff
@@ -103,6 +107,33 @@ def parse_body(self, data):
         raise NotImplementedError()
 
 
+Flag = collections.namedtuple("Flag", ["name", "bit"])
+
+
+class Flags(MutableSet):
```

Comment:
Yeah, let's move it to its own thing. And we can move the tests to their own file too. =)